utiliti/src/de/gurkenlabs/utiliti/swing/AssetPanelItem.java

```diff
@@ -441,18 +445,45 @@ private void exportSpritesheet() {
         chooser.setDialogTitle("Export Spritesheet");
-        if (answer == 0) {
-          XmlExportDialog.export(spriteSheetInfo, "Spritesheet", spriteSheetInfo.getName());
-        } else if (answer == 1) {
-          FileFilter filter = new FileNameExtensionFilter(format.toString() + " - Image", format.toString());
-          chooser.setFileFilter(filter);
-          chooser.addChoosableFileFilter(filter);
-          chooser.setSelectedFile(new File(spriteSheetInfo.getName() + format.toFileExtension()));
-
-          int result = chooser.showSaveDialog(Game.window().getRenderComponent());
-          if (result == JFileChooser.APPROVE_OPTION) {
-            ImageSerializer.saveImage(chooser.getSelectedFile().toString(), sprite.getImage(), format);
-            log.log(Level.INFO, "exported spritesheet {0} to {1}", new Object[] { spriteSheetInfo.getName(), chooser.getSelectedFile() });
+        switch (answer) {
+          case 0: {
+            XmlExportDialog.export(spriteSheetInfo, "Spritesheet", spriteSheetInfo.getName());
+            break;
+          }
+          case 1: {
+            FileFilter filter = new FileNameExtensionFilter(format.toString() + " - Image", format.toString());
+            chooser.setFileFilter(filter);
+            chooser.addChoosableFileFilter(filter);
+            chooser.setSelectedFile(new File(spriteSheetInfo.getName() + format.toFileExtension()));
+            
+            int result = chooser.showSaveDialog(Game.window().getRenderComponent());
+            if (result == JFileChooser.APPROVE_OPTION) {
+              ImageSerializer.saveImage(chooser.getSelectedFile().toString(), sprite.getImage(), format);
+              log.log(Level.INFO, "exported spritesheet {0} to {1}", new Object[]{spriteSheetInfo.getName(), chooser.getSelectedFile()});
+            }
+            break;
+          }
+          case 2: {
+            FileFilter filter = new FileNameExtensionFilter(".json" + " - " + "Spritesheet" + " JSON", "json");
+            chooser.setFileFilter(filter);
+            chooser.addChoosableFileFilter(filter);
+            chooser.setSelectedFile(new File(spriteSheetInfo.getName() + "." + "json"));
+            
+            int result = chooser.showSaveDialog(Game.window().getRenderComponent());
+            if (result == JFileChooser.APPROVE_OPTION) {
+              String fileNameWithExtension = chooser.getSelectedFile().toString();
+              if (!fileNameWithExtension.endsWith(".json")) {
+                fileNameWithExtension += ".json";
+              }
+              String json = AsepriteHandler.exportAnimation(spriteSheetInfo);
+              try (Writer writer = new FileWriter(fileNameWithExtension)) {
+                writer.write(json);
+                log.log(Level.INFO, "Exported {0} {1} to {2}", new Object[]{"Spritesheet", spriteSheetInfo.getName(), fileNameWithExtension});
+              } catch (IOException e) {
+                e.printStackTrace();

```

Comment:
I believe this should be logged instead of using e.printStackTrace()

Reply:
Fixed!